templates/zerver/help/view-the-list-of-installed-bots.md

```diff
@@ -0,0 +1,19 @@
+# View the list of installed bots
+The list of installed bots can only be viewed by an Administrator.
+
+Follow the steps given below to view the list of installed bots using an administrator account:
+
+* Click on the cog (![cog](/static/images/help/cog.png)) icon in the upper right corner
+of the sidebar.
+
+* Choose **Administration** in the panel below:
+
+![admin-panel](/static/images/help/admin.png)
+
+* The following window will appear:
+
+![admin-window](/static/images/help/admin-window.png)
+
+* Click on the tab named **Bots** and you will find the list of installed bots like in the picture below:
+
+![bot-panel-admin](/static/images/help/bot-panel-admin.png)
```

Comment:
@timabbott Sorry but I can't change it because the directory connected to this PR doesn't exist on my cloned repo, You could perhaps merge this and then I'll be able to make changes.